virttest/data_dir.py

```diff
@@ -214,8 +216,16 @@ def get_tmp_dir(public=True):
 
 
 def get_download_dir():
-    if not os.path.isdir(DOWNLOAD_DIR):
-        shutil.copytree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
+    """
+    Returns the dir that holds download definition files, AKA asset files
+
+    Before retuning the download directory, it makes sure the user, writable
+    copy of the download dir is created, and that all asset definition
+    file are in sync.
+
+    :returns: a writable directory populated with asset definition files
+    """
+    dir_util.copy_tree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
```

Comment:
Yep. What would work for me is (ordered):

1. keep it the way it is and document that one needs to remove it (or copy new files manually)
2. override it (interactively) on `--vt-update-config`
3. add new files (but not overwrite nor delete the existing ones)